includes/wikia/nirvana/WikiaSkin.class.php

```diff
@@ -276,15 +276,17 @@ public function getStylesWithCombinedSASS(Array &$sassFiles) {
 		$sassFiles = $this->assetsManager->getSassFilePath($sassFiles);
 
 		// get a single URL to fetch all the required SASS files
-		$sassFilesUrl = $this->assetsManager->getSassesUrl($sassFiles);
+		$sassParams = SassUtil::getSassSettings();
+		$sassParams['crossorigin'] = 'anonymous';
```

Comment:
Not sure how properly to add custom param - feel free to suggest something better :wink: